src/components/global/FormFieldTestWrapper.vue

```diff
@@ -13,8 +13,12 @@
  * - `label` (string): The translation key used for input label in tests.
  * - `name` (string): The name used for id and test selectors.
  * - `options` (array): The options used for radio button tests.
+ * - `required` (boolean): If the input is required.
+ * - `array` (boolean): If the input is an array.
```

Comment:
`array` and `boolean` props should be merge into one props `type`?

Reply:
Updated. Refactored to String type.